udata/core/spatial/api.py

```diff
@@ -48,35 +80,64 @@ def get(self, ids):
         }
 
 
-@ns.route('/zone/<path:id>', endpoint='zone')
-class ZoneAPI(API):
+@ns.route('/zone/<path:id>/children', endpoint='zone_children')
+class ZoneChildrenAPI(API):
     @api.doc('spatial_zone', params={'id': 'A zone identifier'})
-    @api.marshal_with(zone_suggestion_fields)
+    @api.marshal_list_with(feature_collection_fields)
     def get(self, id):
         '''Fetch a zone'''
         zone = GeoZone.objects.get_or_404(id=id)
-        return zone
+        if zone.level_name == 'country':
+            children = zone.regions
+        elif zone.level_name == 'region':
+            children = zone.counties
+        elif zone.level_name == 'county':
+            children = zone.towns
```

Comment:
It's a bit hard for a few reasons:

* for the moment we are in a Russian dolls situation but it'll not be the case anymore once we introduce other levels so adding more abstraction without these edge cases is useless
* even if I use data from the `geo_levels` table, we don't have a strong hierarchy with our current levels (towns' parents doesn't contain counties for instance) so it means that we'll perform too many extra queries
* if I hardcode our French hierarchy it'll not be anymore generic, I think that API will move to `-gouvfr` once we introduce custom levels anyway.

Thoughts?